### PR TITLE
chaindata in sdk

### PR DESCRIPTION
**Pull request by rhlsthrm**
- Pass chainData into SDK
- Pass chaindata properly

## The Problem

Chaindata not passed in through SDK properly.

## The Solution

Pass in chaindata.

## Checklist

- [x] Test manually on `test-ui` using remote chains.
- [ ] Run load tests.
- [ ] Update documentation if needed.
- [ ] Update CHANGELOG.md.

### CODE REVIEW

File: packages/test-ui/src/components/Swap.tsx [hunk 1]
```diff
@@ -590,8 +585,8 @@ export const Swap = ({ web3Provider, signer, chainData }: SwapProps): ReactEleme
                 console.log("changed: ", changed);
               }}
               initialValues={{
-                sendingChain: getChainName(parseInt(Object.keys(swapConfig[selectedPoolIndex].assets)[0]), chainData),
-                receivingChain: getChainName(parseInt(Object.keys(swapConfig[selectedPoolIndex].assets)[1]), chainData),
+                sendingChain: Object.keys(swapConfig[selectedPoolIndex].assets)[0],
+                receivingChain: Object.keys(swapConfig[selectedPoolIndex].assets)[1],
```
Comment (jakekidd) [inline packages/test-ui/src/components/Swap.tsx:589]: no more chain name in the test UI ?

**Review by jakekidd — APPROVED** [review]
lgtm

File: packages/test-ui/src/components/Swap.tsx (hunk 1 above)
Comment (rhlsthrm) [reply packages/test-ui/src/components/Swap.tsx:589]: This was breaking for some reason and I couldn't figure it out. Please help if you can lol.

File: packages/test-ui/src/App.tsx [hunk 2]
```diff
@@ -57,8 +57,8 @@ function App(): ReactElement | null {
 
   useEffect(() => {
     const init = async () => {
-      const json = await utils.fetchJson("https://raw.githubusercontent.com/connext/chaindata/main/crossChain.json");
-      setChainData(json);
+      const data = await getChainData();
+      setChainData(data);
```
Comment (jakekidd) [inline packages/test-ui/src/App.tsx:61]: lol was this wrapped in an init just so we could use await? could just use `.then`

File: packages/test-ui/src/components/Swap.tsx [hunk 3]
```diff
@@ -59,6 +53,7 @@ export const Swap = ({ web3Provider, signer, chainData }: SwapProps): ReactEleme
       setInjectedProviderChainId(chainId);
 
       const sendingChain = form.getFieldValue("sendingChain");
+      console.log('form.getFieldValue("sendingChain"): ', form.getFieldsValue(true));
```
Comment (jakekidd) [inline packages/test-ui/src/components/Swap.tsx:56]: nit: needed?

**Pull request merged**